MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -137,6 +137,13 @@
 #include <setjmp.h>
 #include <unistd.h>
 
+jmp_buf jbAnagram;
+
+void *memset(void *s : byte_count(n), int c, size_t n) : bounds(s, (char *) s + n);
```

Comment:
Can you explain why _FORTIRY_SOURCE flag is required in checked header?
For this flag, checked header is not properly included